File: PHP/Bindings/Phantasma.php
<?php
class PhantasmaAPI { 
	public $host;

	public function __construct($host) {
		$this->host = $host;
	}    

	private function JSONRPC($method, $params) {
		$message = array();
		$message['jsonrpc'] = '2.0';
		$message['method'] = $method;
		$message['params'] = $params;
		$message['id'] ='1';
		$json = json_encode($message);
		$ch = curl_init();
		curl_setopt($ch, CURLOPT_URL, $this->host);
		curl_setopt($ch, CURLOPT_HEADER, false); 
		curl_setopt($ch, CURLOPT_POST, 1);
		curl_setopt($ch, CURLOPT_POSTFIELDS, $json);
		curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
		curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);    
		curl_setopt($ch, CURLOPT_HTTPHEADER, "Accept: application/json");

		$result = curl_exec($ch); 
		curl_close($ch);
		return $result;
	}
    
	{{#each methods}}
	//{{Info.Description}}
	public function {{Info.Name}}({{#each Info.Parameters}}${{Value}}{{#if !@last}}, {{/if}}{{/each}})  
	{	   
		$params = array({{#each Info.Parameters}}${{Value}}{{#if !@last}}, {{/if}}{{/each}});
		return JSONRPC('{{#camel-case Info.Name}}', $params);	
	}
	{{/each}}
} 
?>

File: Go/Bindings/phantasma.go
package phantasma

import (
	"fmt"
	"log"
	"github.com/ybbus/jsonrpc"
)

{{#each types}}
type {{#fix-type Key}} struct 
{
{{#each Value}} {{Name}} {{#if FieldType.IsArray}}[]{{/if}}{{#fix-type FieldType.Name}} `json:"{{Name}}"`{{#new-line}}{{/each}}	   
}
{{/each}}

type phantasma struct {
	host string
	rpcClient jsonrpc.RPCClient
}

func NewAPI(host string) *phantasma {
	fmt.Printf("Connecting to %s\n", host)
    api := new(phantasma)
    api.host = host
    api.rpcClient = jsonrpc.NewClient(host)
    return api
}

{{#each methods}}
//{{Info.Description}}
func (api phantasma) {{Info.Name}}({{#each Info.Parameters}}{{Name}} {{#fix-type Type.Name}}{{#if !@last}}, {{/if}}{{/each}}) {{#if Info.ReturnType.IsArray}}[]{{/if}}*{{#fix-type Info.ReturnType.Name}} {
    var result {{#if Info.ReturnType.IsArray}}[]{{/if}}*{{#fix-type Info.ReturnType.Name}}
    err := api.rpcClient.CallFor(&result, "{{#camel-case Info.Name}}"{{#each Info.Parameters}}, {{Name}}{{/each}})
    if err != nil {
		log.Fatal(err)
    }

	return result
}
{{/each}}